### PR TITLE
[JPRO-63] Scroll bars for axiom section, properties and journal view scroll does not work as expected.

### PR DESCRIPTION
**Pull request by indritbeqiri**
**Current Behavior:**
When the cursor is in a window area, the workspace scroll is activated.

**Expected Behavior:**
When the cursor is in a window area, the workspace scroll should not be activated.

**Steps to Reproduce:**

1. Launch JPro web app.
2. Click on the new journal
3. Click on search icon on the left panel
4. Search for "chronic lung disease" and double click on the concept
5. place the cursor on the first section of the axioms
6. scroll the mouse-wheel until you the scroll reaches the end of first section.
7. scroll the mouse wheel again, and the window moves down to 2nd axiom section. instead of not scrolling.
8. If user continues with the scroll again the focus shifts to the scroll bar on the scalable desktop.

### CODE REVIEW

File: kview/src/main/java/dev/ikm/komet/kview/mvvm/view/details/DetailsController.java [hunk 1]
```diff
@@ -337,7 +331,7 @@ public void initialize() {
             for (PatternEntityVersion pattern : patterns) {
                 MenuItem menuItem = new MenuItem();
                 if (isInMembershipPattern(currentConceptFacade.nid(), pattern.nid(), viewCalculator)) {
-                //if (isInMembershipPattern(currentConceptFacade.nid(), pattern.nid())) {
+                    //if (isInMembershipPattern(currentConceptFacade.nid(), pattern.nid())) {
```
Comment (carldea) [inline kview/src/main/java/dev/ikm/komet/kview/mvvm/view/details/DetailsController.java:334]: Do we want to remove unused comment?

File: kview/src/main/java/dev/ikm/komet/kview/mvvm/view/details/DetailsController.java [hunk 2]
```diff
@@ -515,7 +509,7 @@ private ContextMenu buildMenuOptionContextMenu() {
         Object[][] menuItems;
         // show the 'Add Fully Qualified' option when it is a new concept in create mode and there is no fully qualified name
         if (this.conceptViewModel.getPropertyValue(MODE).equals(CREATE) &&
-            (getConceptViewModel().getPropertyValue(FULLY_QUALIFIED_NAME) == null)) {
+                (getConceptViewModel().getPropertyValue(FULLY_QUALIFIED_NAME) == null)) {
```
Comment (carldea) [inline kview/src/main/java/dev/ikm/komet/kview/mvvm/view/details/DetailsController.java:512]: **NOTE:** I may have to fix(revisit) Cognitive's `getProperty(...)` method to not return type `T` but instead `Property<T>`.
That way you could do the following:
```java
 .getProperty("someProp").isNull()
```

Comment (carldea) [reply kview/src/main/java/dev/ikm/komet/kview/mvvm/view/details/DetailsController.java:512]: @indritbeqiri  Nothing you need to do. I just wanted to point that out. My intent is to improve the mvvm api.

**Review by carldea — APPROVED** [review]
Looks good to me! Just hope you got a chance to see my notes.

File: kview/src/main/java/dev/ikm/komet/kview/mvvm/view/details/DetailsController.java (hunk 2 above)
Comment (besidev) [reply kview/src/main/java/dev/ikm/komet/kview/mvvm/view/details/DetailsController.java:511]: Thanks for pointing that out, Carl. We totally agree that switching to `Property<T>` would make the MVVM API more consistent and flexible.

@besidev @indritbeqiri

File: kview/src/main/java/dev/ikm/komet/kview/mvvm/view/details/DetailsController.java (hunk 1 above)
Comment (indritbeqiri) [reply kview/src/main/java/dev/ikm/komet/kview/mvvm/view/details/DetailsController.java]: Unused comment has been removed.

**Review by carldea — APPROVED** [review]
Sorry for the slow reply on the PR. I will be merging it now.

**Comment by indritbeqiri** [comment]
No worries, Carl! Thank you for merging the PR.

**Pull request merged**